output/mapping/eforms/guidance.yaml

```diff
@@ -1746,9 +1746,9 @@
     Business groups:
         BG-77: Modification
     eForms guidance: Discard. The notice and sections that were modified can be determined by comparing previous releases with the current release.
-    eForms example: <efac:ContractModification><efac:Change><efbc:ChangedSectionIdentifier>LOT-0002</efbc:ChangedSectionIdentifier></efac:Change></efac:ContractModification>
+    eForms example: <efac:ContractModification><efbc:ChangedNoticeIdentifier schemeName="notice-id-ref">755db7f6-6aa1-453b-9087-1df5f7725112-02</efbc:ChangedNoticeIdentifier></efac:ContractModification>
     OCDS example: ''
-    sdk: https://docs.ted.europa.eu/eforms/latest/schema/contract-modification-notice.html
+    sdk: https://docs.ted.europa.eu/eforms/latest/schema/all-in-one.html#contractModificationNoticeSection
```

Comment:
FYI, don't use all-in-one, as the eForms maintainers are not fond of it and might remove it: https://github.com/OP-TED/eForms-SDK/discussions/210

Reply:
I've updated all links accordingly.

Reply:
ah, good to know, I'll note that for next time